.github/scripts/ubuntu-20.04/download_build.sh

```diff
@@ -0,0 +1,22 @@
+# Download installers for compiled dependencies
+
+mkdir -p ${DOWNLOAD_PATH}
+
+## Thrift 0.16
+
+wget -O ${DOWNLOAD_PATH}/thrift-0.16.0.tar.gz "http://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=thrift/0.16.0/thrift-0.16.0.tar.gz"
```

Comment:
In the Gitlab CI, we use the `http://archive.apache.org/dist/thrift/0.16.0/thrift-0.16.0.tar.gz` URL.

Reply:
Fixed.